packages/xo-web/src/xo-app/pool/tab-patches.js

```diff
@@ -197,12 +218,18 @@ export default class TabPatches extends Component {
               {ROLLING_POOL_UPDATES_AVAILABLE && (
                 <TabButton
                   btnStyle='primary'
-                  disabled={isEmpty(missingPatches) || isSingleHost}
+                  disabled={isEmpty(missingPatches) || this.nVmsRunningOnLocalStorage() > 0}
                   handler={rollingPoolUpdate}
                   handlerParam={pool.id}
                   icon='pool-rolling-update'
                   labelId='rollingPoolUpdate'
-                  tooltip={isSingleHost ? _('multiHostPoolUpdate') : undefined}
+                  tooltip={
+                    this.nVmsRunningOnLocalStorage() > 0
+                      ? _('nVmsRunningOnLocalStorage', {
+                          nVms: this.nVmsRunningOnLocalStorage(),
+                        })
+                      : undefined
+                  }
```

Comment:
Why did you remove `isSingleHost` here? Isn't that supposed to be an additional condition?

Reply:
so it is a bit confusing but this PR comes with [this one](https://github.com/vatesfr/xen-orchestra/pull/7286/files), it was a single task divided in 2 PRs, so 'isSingleHost' is still here, but it is a condition in the other PR who's already been merged